CONTRIBUTING.md

```diff
@@ -29,6 +31,16 @@ Console](https://console.cloud.google.com/apis/library).
 
         npm test
 
+### Running the tests for a Pull Request
+
+When a Pull Request is opened, reopened, or has new commits pushed the tests will be run.
+
+If the tests for a sample change do not run, they can be triggered by adding the `actions:force-run` label.
+
+If tests need to be triggered a second time, manually remove and re-add this label.
```

Comment:
If you need tests to re-run multiple times (merging changes from main, updating changes based on feedback), manually remove the 'actions:force-run' and then re-add this label. 

(not just a second time, but any number more than 1, is there an amount of time that needs to be waited in between manually removing and then readding for it to get caught?)

Reply:
I've converted your feedback to a suggested change below, but to address a couple questions:

1. I've not noticed any particular wait time, but I usually wait for the label updated to be recorded in the comment/timeline view by GitHub.

2. If you merge changes from main the PR will automatically retest. The only times to use this are queueing is stuck or we think the failure was a flake or external bug.

```suggestion
If tests need to be triggered multiple times, manually remove `actions:force-run` and then re-add this label.
```